metrotranscom/doorway
# Fix/change jurisdiction notification url & add Terms of Service

# Pull Request Template

## Issue Overview
The notification URL was linked within the code base, rather than pulling the value from the db. Seed data has been updated with the correct GovDelivery link and changed code that references that URL.

TODO: Need to update ListingsCombined & rebase

Link is: https://public.govdelivery.com/accounts/CAMTC/signup/36832

This PR addresses #issue



## How Can This Be Tested/Reviewed?
Reseed data using yarn db:reseed`, then `yarn install` & `yarn dev` on the public site and verify that the link on the Listings page links to the correct govDelivery site.

go to `http://localhost:3100/jurisdictions` and see that the `notificationsSignUpURL` URL is now https://public.govdelivery.com/accounts/CAMTC/signup/36832

![image](https://github.com/metrotranscom/doorway/assets/12400715/1243e03d-8d7c-428f-aba1-0a207a72ed97)




## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
